plugins/repository-azure/src/main/java/org/opensearch/repositories/azure/AzureStorageSettings.java

```diff
@@ -105,6 +105,42 @@
         () -> KEY_SETTING
     );
 
+    // See please NettyAsyncHttpClientBuilder#DEFAULT_CONNECT_TIMEOUT
+    public static final AffixSetting<TimeValue> CONNECTION_TIMEOUT_SETTING = Setting.affixKeySetting(
+        AZURE_CLIENT_PREFIX_KEY,
+        "connection.timeout",
```

Comment:
Minor, but what do you think about using "connect" instead of "connection" for this setting and the variable names to keep it consistent with the underlying Azure property?

Reply:
@andrross thank you, sure, no objections, renaming